grow/testing/testdata/extract_localized/localized_pod/content/localized_collection/doc_in_parts.html

```diff
@@ -0,0 +1,22 @@
+---
+$localization:
+  locales:
+  - it
+  - ja
+  - ko
```

Comment:
Prior to me adding `ko`, `foo@` in the base part was actually (correctly) overridden by all of the localized document parts. By adding `ko` and adjusting the tests, we verify `foo@` in the base part is extracted to the KO catalog and that `foo@` from the base part is *not* extracted to localized catalogs (since it is overridden, only the localized `foo@`s are extracted). @stucox I think this is correct?